worknenjoy/gitpay
# Fix/slow site

## Description

> Spliting `app.js` bundle for better performance and caching. 

## Changes

- Moved `frontend/public/index.html` to `frontend/src/index.html`
- Changed webpack config files and server.js

## Issue

> #450 

## Impacted Area

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- You can check in developer menu. The size should have decreased.

## Before
> Screenshot from the state before

## After
> Screenshot from the state after your Pull Request
